lib/heroku-deflater/cache_control_manager.rb

```diff
@@ -20,9 +20,9 @@ def setup_max_age(max_age)
 
     def cache_control_headers
       if rails_version_5?
-        { 'Cache-Control' => cache_control }
+        { 'Cache-Control' => app.config.public_file_server.headers }
```

Comment:
@bmclean I've verified your solution in Rails 5. Another version of line 23 that works is:
`{ 'Cache-Control' => app.config.public_file_server.headers['Cache-Control'] }`

Your solution has the potential to include multiple headers (should any be present in `app.config.public_file_server.headers`), whereas the line above will only ever set the `Cache-Control` header.

I'm honestly not sure which is preferable. It seems like if any other headers are set in Rails they should be preserved by the middleware, yet the intent of the method appears to be to set only the `Cache-Control` header, and served_zipped_assets.rb appears not to preserve app headers by any other means.

So, we have a fix for this ready as soon as we figure out which method is best.